modules/config.xqm

```diff
@@ -57,6 +61,14 @@ declare variable $config:odd-transform-default := function($xml, $parameters) {
 
 declare variable $config:module-config := doc($config:odd-source || "/configuration.xml")/*;
 
+(:
+ : The default to use for determining the amount of content to be shown
+ : on a single page. Possible values: 'div' for showing entire divs (see
+ : the parameters below for further configuration), or 'page' to browse
+ : a document by actual pages determined by TEI pb elements.
+ :)
+declare variable $config:default-view := "body";
```

Comment:
I'd like to understand why 'body' rather than 'div' - I would expect the latter to be more useful for e.g. frus sub-pages, but perhaps there is more TEI content for smaller chunks of content that I'm missing?

Reply:
Good point, will change this to "div". We can easily change it back if it doesn't work out.